### PR TITLE
[Subpart] Clean dependencies - minor changes only

### PR DESCRIPTION
**Pull request by chibenwa**
Subpart of https://github.com/linagora/james-project/pull/2409

Before:

```
benwa@horizon ../guice/cassandra-rabbitmq-guice/target (git)-[master] % ls -l james-server-cassandra-rabbitmq-guice.lib/ | wc -l
535
benwa@horizon ../guice/cassandra-rabbitmq-guice/target (git)-[master] % du -h -d 1 james-server-cassandra-rabbitmq-guice.lib 
231M    james-server-cassandra-rabbitmq-guice.lib
```

After:

```
benwa@horizon ../guice/cassandra-rabbitmq-guice/target (git)-[JAMES-2671-fix-dep-simple] % ls james-server-cassandra-rabbitmq-guice.lib/ -l | wc -l
359
benwa@horizon ../guice/cassandra-rabbitmq-guice/target (git)-[JAMES-2671-fix-dep-simple] % du -h -d 1 james-server-cassandra-rabbitmq-guice.lib
116M    james-server-cassandra-rabbitmq-guice.lib
```

### CODE REVIEW

File: server/blob/blob-objectstorage/pom.xml [hunk 1]
```diff
@@ -140,6 +140,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```
Comment (rouazana) [inline server/blob/blob-objectstorage/pom.xml]: should be before the previous one?

Comment (chibenwa) [reply server/blob/blob-objectstorage/pom.xml]: Well, slf4j api is still used in our code while the aws-3 lib requires the jcl-over-slf4j

Comment (rouazana) [reply server/blob/blob-objectstorage/pom.xml]: just talking about alphabetic order in the pom

Comment (chibenwa) [reply server/blob/blob-objectstorage/pom.xml]: Ahhhh

**Pull request closed without merging**